src/storage.ts

```diff
@@ -417,6 +429,13 @@ export class Storage extends Service {
     this.getHmacKeysStream = paginator.streamify('getHmacKeys');
   }
 
+  private static sanitizeEndpoint(url: string) {
```

Comment:
This should be tested in its own "describe('sanitizeEndpoint')". (You could probably just copy gcs-resumable-upload's)